eng/tools/mcp/azure-sdk-python-mcp/main.py

```diff
@@ -196,22 +196,28 @@ def tox_tool(package_path: str, environment: str, repo_path: str, tox_ini_path:
     return run_command(command, cwd=repo_path)
 
 @mcp.tool("init")
-def init_tool(tsp_config_url: str, repo_path: str) -> Dict[str, Any]:
-    """Initializes and generates a typespec client library directory given the url.
+def init_tool(tsp_config_path: str, repo_path: str, is_local: bool = False) -> Dict[str, Any]:
```

Comment:
According to description `param bool is_local: Whether the tsp_config_path is a local file path (True) or a URL (False)`, we could judge `is_local` from tsp_config_path so I think it is not necessary in the signature.